src/multi_vector_simulator/E1_process_results.py

```diff
@@ -942,3 +944,29 @@ def get_units_of_cost_matrix_entries(dict_economic, kpi_list):
         else:
             unit_list.append(kpi_cost_unit_dict[key])
     return unit_list
+
+
+def get_emissions(dict_asset):
```

Comment:
How about: `calculate_emissions_from_flow`? 
Do you think it would make sense to move this to E3? IMO, in E1, there should be rather big generic functions and the equations should be in E3.

Reply:
I wasn't aware of that. Apart from that, E3 is called indicator calculation, which fits as well. Will move and rename it.